en/docs/learn/examples/protocol-switching/switching-from-jms-to-http.md

```diff
@@ -6,35 +6,42 @@ This example demonstrates how the Micro Integrator receives a messages over the
 
 Following are the integration artifacts (proxy service) that we can used to implement this scenario. See the instructions on how to [build and run](#build-and-run) this example.
 
```

Comment:
receives a messages -> receives messages

Reply:
returns the reply back to the client -> returns the reply to the client

Reply:
Following are the integration artifacts (proxy service) that we can used to implement this scenario.
->

Following are the integration artifacts (proxy service) that we can use to implement this scenario.